docs/commands/ft.search.md

```diff
@@ -76,7 +76,9 @@ Searches the index with a textual query, returning either documents or just ids.
 
 - **TIMEOUT {milliseconds}**: If set, we will override the timeout parameter of the module.
 
-* **PARAMS {nargs} {name} {value}**. Define one or more value parameters. Each parameter has a name and a value. Parameters can be referenced in the **query** by a `$`, followed by the parameter name, e.g., `$user`, and each such reference in the search query to a parameter name is substituted by the corresponding parameter value. For example, with parameter definition `PARAMS 4 lon 29.69465 lat 34.95126`, the expression `@loc:[$lon $lat 10 km]` would be evaluated to `@loc:[29.69465 34.95126 10 km]`. Parameters cannot be referenced in the query string where concrete values are not allowed, such as in field names, e.g., `@loc`
+* **PARAMS {nargs} {name} {value}**. Define one or more value parameters. Each parameter has a name and a value. Parameters can be referenced in the **query** by a `$`, followed by the parameter name, e.g., `$user`, and each such reference in the search query to a parameter name is substituted by the corresponding parameter value. For example, with parameter definition `PARAMS 4 lon 29.69465 lat 34.95126`, the expression `@loc:[$lon $lat 10 km]` would be evaluated to `@loc:[29.69465 34.95126 10 km]`. Parameters cannot be referenced in the query string where concrete values are not allowed, such as in field names, e.g., `@loc`. To use `PARAMS`, `DIALECT` must be set to 2.
+
+- **DIALECT**: The query syntax to use for the query. Options are 1 and 2.
```

Comment:
These additions seem to have been lost during the move of the documentation to redis.io

https://github.com/RediSearch/RediSearch/pull/2671/files#diff-13483592b824255998c18c26ba13d15aaa8cf9aac19d0135e07000204a0cbaabR462

Also relevant in `FT.AGGREGATE`, `FT.EXPLAIN`, `FT.EXPLAINCLI`, `FT.SPELLCHECK`